test/linalg/spectral.jl

```diff
@@ -9,156 +9,158 @@ full(nbt::Nonbacktracking) = full(sparse(nbt))
     g4 = PathDiGraph(5)
     g5 = DiGraph(4)
     for g in testgraphs(g3)
```

Comment:
If this is just whitespace, can it be left out of this PR? For the sake of clarity.